snmp/datadog_checks/snmp/data/default_profiles/dell-sonicwall.yaml

```diff
@@ -13,21 +13,21 @@ metadata:
           MIB: SONICWALL-COMMON-MIB
           OID: 1.3.6.1.4.1.8741.2.1.1.2.0
           name: snwlSysSerialNumber
+#      TODO:
+#        symbol:
+#          MIB: SONICWALL-COMMON-MIB
+#          OID: 1.3.6.1.4.1.8741.2.1.1.4.0
+#          name: snwlSysROMVersion
```

Comment:
Not sure if we should delete it completely, so I commented this out

Reply:
I think it's probably fine to delete it :)